### PR TITLE
Introduction added

### PR DESCRIPTION
**Pull request by avdyushin**

### CODE REVIEW

**Review by joopst — APPROVED** [review]
nice, this really helps

File: README.md [hunk 1]
```diff
@@ -2,10 +2,31 @@
 
 # State management in iOS
 
+## Introduction
+
+In iOS development it's always good to choose a right architecture.
+As you know in projects used `MVC` pattern it becomes hard
```
Comment (oradyvan) [inline README.md:8]: Would be "As you know in projects that are using `MVC` pattern it becomes hard"

File: README.md [hunk 2]
```diff
@@ -2,10 +2,31 @@
...
+to maintain and test massive view controller after some time.
+
+Let's try to make it more clean and reusable using one way data flow architecture.
```
Comment (oradyvan) [inline README.md:11]: Maybe change "one way data flow architecture" to "unidirectional data flow pattern" as you mention it below.

File: README.md [hunk 3]
```diff
@@ -2,10 +2,31 @@
...
+Main aspects:
+
+* The whole application `state` is stored in single object (store)
+* The only way to change the `state` is to trigger (dispatch) an `action` (an object describing what to do)
+* `Reducer` specifies how to generate new `state` based on given `action` and current `state`.
+
+That's basically it.
+
+In this tutorial you will see how to build sample iOS application
+using exiting unidirectional framework.
```
Comment (oradyvan) [inline README.md:24]: I would say "using `ReSwift` framework that provides unidirectional data flow pattern implementation.

File: README.md [hunk 4]
```diff
@@ -2,10 +2,31 @@
...
+
 ## About application
 
-We are going to build universal application to allow users track they frequently data.
-It could be let's say mileage each month or outcome each day.
+We are going to build simple application to allow users track they frequently data.
```
Comment (oradyvan) [inline README.md:28]: I would say "We are going to build simple application that allows users track their frequently collected data."

**Review by oradyvan — COMMENTED** [review]
Some comments on the wording and typos.

**Comment by avdyushin** [comment]
Thanks for comments! Will fix in next pull request

**Pull request merged**